bulletin/index.html

```diff
@@ -203,6 +203,53 @@ <h2 class="featured-article__title"> Engineering Robust Learning Environments Th
+			</div>
+			<div class="mention-content">
+				<blockquote class="mention-content__item"> I learned many examples of good behavior and about capitalization.</blockquote>	
+			</div>
+			<div class="mention-content">
+				<blockquote class="mention-content__item">I learned a lot about bridges, solar energy, mathematics, and other things. I also learned that I should not speak with a loud voice and not to disturb others. I also learned that working on a team is extremely important to achieve my goals.
+				</blockquote>
+			</div>
+			<div class="mention-content">
+				<blockquote class="mention-content__item"> I learned that I should ask questions when in doubt. If I remain silent, I will never learn what I need to learn. I also learned some mathematics and Spanish. I did not know how to write an essay. I used to make a textual copy of the material.</blockquote>
+			</div>
+			<div class="mention-section">
+				<h3 class="mention-section__subtitle">Acknowledgments</h3>
+			</div>
+			<div class="mention-content">
+				<p class="mention-content__item mention-content__item_xp"><a class="inner-link--yellow" href="https://buffalotungsten.com/" target="_blank">Buffalo Tungsten, Inc.</a>, a private U.S. corporation, is the principal supporter of MACILE, an education program of Complex Systems Optimization Laboratory. </p>
+			</div>
+			<div class="mention-content">
+				<p class="mention-content__item mention-content__item_xp">The Language and Grammar Program was generously written for MACILE by <a class="inner-link--yellow" href="https://www.marlboro.edu/live/profiles/15-rosario-de-swanson" target="_blank">Rosario M. de Swanson, Ph.D.</a>, Professor of Language and Literature at Marlboro College, U.S.A.</p>
+			</div>
+			<div class="mention-content">
+				<p class="mention-content__item mention-content__item_xp"><cite>Los Indios de Quisqueya</cite>, history book of the Tainos in the Hispaniola, and foundation of the MACILE Spanish curriculum, was donated by the <span style="color: #d24b4d">Tavares Foundation.</span> The author, Juan Tómas Tavares K., permitted its use without reservations.</p>
+			</div>
+			<div class="mention-content">
+				<p class="mention-content__item mention-content__item_xp"> Dominican teachers, school directors, and school secretaries made the 2007 and 2008 MACILE Summer Academy possible. These remarkable women volunteered to teach and to receive the required training. They promoted MACILE among the schools, encouraged students to apply, motivated other teachers to participate, secured space for the academy, and provided administrative support. The teachers are Yahaira Soriano from Itabo, Antonia Asencio, and Gladys Vizcaíno from Colegio Padre Zegrí. Directors Bernarda N Lorenzo, Itabo Elementary School, and Sor Isabel Jiménez, Colegio Padre Zegrí ensured a home for MACILE in Itabo and San Gregorio de Nigua.  Administrative assistant Sandra Pinales kept meticulous account of all the records.	</p>
```

Comment:
We don't need to fix this now, but I find this section a bit redundant with the content of the bulletin, no? Is the idea to draw it out a bit more?

Reply:
Yes, the idea is to draw it out a bit more, including a bit more of information. But I agree is a bit redundant, as well.